apps/common/behaviours/casework-submission.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
 
+const AuthToken = require('../models/auth-token');
 const CaseworkModel = require('../models/i-casework');
 const StatsD = require('hot-shots');
 const client = new StatsD();
 
 const Compose = func => superclass => class extends superclass {
   prepare() {
-    if (typeof func === 'function') {
```

Comment:
func is always a function which is essentially creates a payload object to send to icasework on all 4 journeys.